src/controllers/orderController.js

```diff
@@ -79,6 +104,17 @@ const getSingleOrder = async (req, res) => {
   checkPermissions(req.user, order.user);
 
   res.status(StatusCodes.OK).json({ order });
+  /*
+  #swagger.summary = 'Fetch an order by id'
+  #swagger.parameters['id'] = {
+    description: 'Mongo ObjectID of the order to fetch',
+  }
+  #swagger.responses[200] = {
+    description: 'Order successfully fetched.',
+    schema: { $ref: '#/definitions/Order' } 
```

Comment:
when the response is returned; it's nested inside an object like this: `{ "order": <order object as defined in the schema> }`. But out swagger doc here just says it will return `{ <order object> }` without that top-level key:

Real response:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/3039da6d-dacd-4343-bbf7-1c78e0ae7444)

vs. documented response:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/fd029933-1950-41ca-a2b8-17dfe8c49323)

so we need to make the documentation match by wrapping our schema:

```suggestion
    schema: { order: { $ref: '#/definitions/Order' } }
```